wolfssl/internal.h

```diff
@@ -1846,6 +1845,8 @@ enum Misc {
     READ_PROTO         = 0     /* reading a protocol message */
 };
 
+#define MAX_RECORD_SIZE 16384  /* 2^14, max size by standard */
```

Comment:
Should this go near `#ifdef RECORD_SIZE` at line 2276? Making this a macro is fine with me. I don't see any downstream issues.